apps/api/src/applicant/external-api.service.ts

```diff
@@ -628,16 +631,26 @@ export class ExternalAPIService {
     }
   }
 
-  async getApplicants() {
-    return this.ienapplicantRepository.find({
-      relations: [
-        'status',
-        'added_by',
-        'updated_by',
-        'jobs',
-        'applicant_status_audit',
-        'applicant_audit',
-      ],
-    });
+
+  
+  async getApplicants(start?:string,end?:string ) {
+      return this.ienapplicantRepository.find({
+        where: {
+          updated_date: Between(
+            new Date(start ||'1918-07-18' ).toISOString(),
```

Comment:
Does this date have special meaning?

Reply:
It's the day World war 1 was declared. I needed an arbitrary date that should be well before any record is updated. 
It's an easter egg, and I can remove it if you like.